Solution1/DataAccessLayer/Repositories/WeatherRepository.cs

```diff
@@ -0,0 +1,24 @@
+﻿using DataAccessLayer.Configuration;
+using DataAccessLayer.Models;
+using DataAccessLayer.Repositories.Abstract;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace DataAccessLayer.Repositories
+{
+    public class WeatherRepository : IWeatherRepository
+    {
+        readonly RepositoryContext _repository;
+
+        public WeatherRepository(RepositoryContext repository)
```

Comment:
WeatherRepository is repository, RepositoryContext is context, not repository

Reply:
+